tests/disabled_hypothesis/test_product_finder_hypothesis.py

```
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# flake8: noqa: W503,W504
+# ruff: noqa: E501
```

Ruff (0.14.1)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)